src/bridge/cockpitmountsamples.c

```diff
@@ -57,9 +57,9 @@ cockpit_mount_samples (CockpitSamples *samples)
       if (strlen (line) == 0)
         continue;
 
-      /* Only look at real devices
+      /* Only look at real devices and remote mounts
        */
-      if (line[0] != '/')
+      if (line[0] != '/' && strchr(line, ':') == NULL)
```

Comment:
I wonder if it is safe to stat remote file systems like this once or even twice (not sure about the frequency of our sampling) a second?

Reply:
We sample once a second.  What might go wrong if we sample too much?

Reply:
I'm not sure how much overhead that is. Imagine having a dozen NFS mounts which might not be super-fast, and sampling all of them once per second. That could create some network noise/latencies or even overlapping? E. g. nautilus/gvfs also doesn't continuously sample remote file systems, only on demand. So checking `df` on remote file systems once seems like an acceptable compromise?

Don't get me wrong, I don't veto this, but I wanted to point this out as a potential problem.

Reply:
Yes, I think I am convinced.

Reply:
Fixed.  Now we poll only once per fstab change.